olympus-oir/read_oir_info.py

```diff
@@ -126,7 +121,68 @@ def main(filepath):
     nTLoop = nTLoop or 1
     print(nLLoop, nTLoop, nZLoop)
 
+    # ====================
+    # Output
+    # ====================
+
+    result_data = {
```

Comment:
Please respond to the following TODOs (4 points)

Reply:
@itutu-tienday 
I'm fixing the first 3 TODOs. About the 4th ToDo (ByteDepthCh0), I wanna ask your opinion. In the cpp source, method CChannelInfo::GetDepthOfCh0_TM return double, but the data type of the source is Integer. I don't know why they have to cast it to double.
=> Do you want to use integer, or cast it to float (in python we don't have double) follow the cpp source?

Reply:
@itutu-tienday 
I've created PR #3 , please check it.

Reply:
@tuanna26 
> => Do you want to use integer, or cast it to float (in python we don't have double) follow the cpp source?

I am not sure of the intent of CChannelInfo::GetDepthOfCh0_TM returning double, but as you pointed out, the original value is an int, so please leave it as an int on the python side.
(The actual output to .mat also seems to be an int.)